framework/doc/content/source/postprocessors/InterfaceIntegralVariableAveragePostprocessor.md

```diff
@@ -0,0 +1,21 @@
+# InterfaceIntegralVariableAveragePostprocessor
+
+## IMPORTANT NOTE
+*** Any derived class of InterfaceUserObject does not support yet getMaterialProperty, please use auxvariables ***
```

Comment:
I didn't follow all the history of your implementation. Can you summarize why `getMaterialProperty` can't be called in this UO? I have a class deriving from it and calling `getMaterialProperty` and `getNeighborMaterialProperty`, and at first glance I didn't see any problem with that.

Reply:
@vincentlaboure `getMaterialProperty` should trigger the calculation of material property. However, in the loops `ComputeMaterialThreads:OnInterface` is not defined yet therefore there is no guarantee you will get the most update material property when executing the UO. You should not see differences if you execute the UO with `execute_on=TIMESTEP_END` but you should see bad results if you use `execute_on=LINEAR`.  This is the reason why you should not use `getMaterialProperty` from an `InterfaceUO` until PR #12101 is completed and merged. The workaround to this is to dump MP into `AuxVars` and use them inside the `InterfaceUO`.
You should be able to check this bad behaviour easily. let me know if you try and it works :)